app/helpers/summaries_helper.rb

```diff
@@ -23,18 +23,36 @@ def get_summaries_table_info(assignment, grader_id=nil)
         end
     end
 
+    parts = groupings.select &:has_submission?
+    results = Result.where(submission_id:
+                             parts.map(&:current_submission_used))
+                    .order(:id)
     groupings.map do |grouping|
+      submission = grouping.current_submission_used
+      if submission.nil?
+        result = nil
+      elsif !submission.remark_submitted?
+        result = (results.select do |r|
+                   r.submission_id == submission.id
+                 end).first
```

Comment:
`end` at 37, 17 is not aligned with `results.select do |r|` at 35, 18 or `result = (results.select do |r|` at 35, 8